bool enable
int32 power  # 0-100%
---
bool success
string message 